package/firmware/ipq-wifi/Makefile

```diff
@@ -27,28 +27,34 @@ endef
 ALLWIFIBOARDS:= \
 	aruba_ap-365 \
 	devolo_magic-2-wifi-next \
+	dynalink_dl-wrx36 \
+	edgecore_eap102 \
 	edgecore_ecw5410 \
 	edgecore_oap100 \
+	edimax_cax1800 \
 	extreme-networks_ws-ap3915i \
 	glinet_gl-a1300 \
 	glinet_gl-ap1300 \
 	glinet_gl-s1300 \
 	linksys_ea8300 \
 	p2w_r619ac \
 	pakedge_wr-1 \
+	qnap_301w \
 	qxwlan_e2600ac-c1 \
 	qxwlan_e2600ac-c2 \
+    redmi_ax6 \
 	sony_ncp-hg100-cellular \
 	teltonika_rutx \
-	zte_mf289f
```

Comment:
Is the removal of the ZTE MF289F intentional?

Reply:
I'm guessing not since `generate-ipq-wifi-package` is called for that target too.

Reply:
Nope, it happened during numerous rebases, will respin after CI completes